edith-step/src/UserChart.test.js

```diff
@@ -1,42 +1,46 @@
+import { enableFetchMocks } from 'jest-fetch-mock'
+enableFetchMocks();
+import {mount} from 'enzyme';
 import React from 'react';
-import { mount, unmount } from 'enzyme';
 import ReceiptInput from './ReceiptInput';
-import axios from 'axios';
-jest.mock('axios');
 import './setupTests.js';
 import App from './App';
-import UserChart from "./UserChart";
-
-it('UserChart should be mounted to App', () => {
-  HTMLCanvasElement.prototype.getContext = jest.fn()
-  jest.mock('react-chartjs-2', () => ({
-      UserChart: () => null,
-    }))
-  axios.get.mockResolvedValue({
-      data: {
-        labels: [],
-        datasets: [
-          {
-            label: "Weeek Total",
-            data: [],
-            backgroundColor: ["rgba(75, 192, 192, 0.6)"],
-            borderWidth: 4
-          }
-        ]
-      }
-  });  
-  const component = mount(<App />);
-  expect(component.contains(<UserChart />)).toBeTruthy();
-});
+import LineChart, {retrieveData, BarGraph, DoughnutChart} from "./UserChart";
+
+
+const component = mount(<App />);
 
-it('Axios test', () => {
-  axios.get.mockResolvedValue({
-      data: {
-          items: ["item1"]
-      } 
+it('Should receive the correct week data from retrieveData', () => {
+//   HTMLCanvasElement.prototype.getContext = jest.fn()
+//   jest.mock('react-chartjs-2', () => ({
+//       UserChart: () => null
+//     }));
```

Comment:
can this commented code (and on test below) be deleted?

Reply:
Yea!